src/core/mac/wakeup_tx_scheduler.cpp

```diff
@@ -88,7 +88,7 @@ Mac::TxFrame *WakeupTxScheduler::PrepareWakeupFrame(Mac::TxFrames &aTxFrames)
 
     target.SetExtended(mWedAddress);
     source.SetExtended(Get<Mac::Mac>().GetExtAddress());
-    radioTxUs = static_cast<uint32_t>(Get<Radio>().GetNow()) + (mTxTimeUs - TimerMicro::GetNow());
+    radioTxUs = mTxTimeUs - TimerMicro::GetNow();
```

Comment:
Also suggest maybe renaming it to `radioTxDelay` (since now it is relative delay?).